spec/models/events/creator_spec.rb

```diff
@@ -1,6 +1,11 @@
 require 'rails_helper'
 
 RSpec.describe Events::Creator, :type => :model do
+
+  before do
+    Timecop.freeze(Time.local(1990))
```

Comment:
I see now why you are setting to a point in time (removes the precision problem).

Would you consider switching this to `expect(x.start_at).to be_within(1).of(start_at)` and removing Timecop?